test/Microsoft.AspNetCore.SignalR.Client.Tests/ResponseUtils.cs

```diff
@@ -6,9 +6,9 @@
 using System.Net;
 using System.Net.Http;
 using Microsoft.AspNetCore.Connections;
-using Microsoft.AspNetCore.Sockets;
 using Newtonsoft.Json;
-using SocketsTransportType = Microsoft.AspNetCore.Sockets.TransportType;
+
```

Comment:
nit: blank line

Reply:
We haven't been terribly consistent about this but I was originally thinking that type alias `using`s should be separated from standard namespace-import `using`s.